### ISSUE TITLE
Hyper-parameters used for reproducibility

### ISSUE THREAD

**Issue by dmgcsilva** [EvolvingLMMs-Lab/LLaVA-OneVision-1.5#109]
Hello, thanks for the great work and contribution to the community!

Would it be possible to share more details regarding the hyperparameters used for each stage to ensure reproducibility?
For example the batch size, max sequence length, learning rate and scheduler, warmup ration, etc.

Many thanks!

**Comment by yiyexy** [comment]
Sure, you can find the scripts for each stage in the repo. We used the default values for training.
